cmd/cluster-agent/api/agent/agent.go

```diff
@@ -43,6 +46,9 @@ func SetupHandlers(r *mux.Router) {
 	r.HandleFunc("/config/list-runtime", settingshttp.Server.ListConfigurable).Methods("GET")
 	r.HandleFunc("/config/{setting}", settingshttp.Server.GetValue).Methods("GET")
 	r.HandleFunc("/config/{setting}", settingshttp.Server.SetValue).Methods("POST")
+	r.HandleFunc("/tagger-list", getTaggerList).Methods("GET")
+	r.HandleFunc("/workload-list/short", getShortWorkloadList).Methods("GET")
+	r.HandleFunc("/workload-list/verbose", getVerboseWorkloadList).Methods("GET")
```

Comment:
That’s really a tiny point and maybe a mater of preference, but I would have more probably passed the verbosity as a query parameter rather than introducing two routes.
Something like `/workload-list?verbose=true`
The advantage is that it would be easier to implement several levels of verbosity for ex.

```suggestion
	r.HandleFunc("/workload-list", getWorkloadList).Methods("GET")
```